compiler/damlc/daml-opts/daml-opts-types/DA/Daml/Options/Types.hs

```diff
@@ -55,6 +55,10 @@ data Options = Options
     -- ^ Controls whether the scenario service server runs all checks
     -- or only a subset of them. This is mostly used to run additional
     -- checks on CI while keeping the IDE fast.
+  , optHlintEnabled :: Bool
```

Comment:
It looks like you are already switching on `optHlintDataDir` so this option seems to be redundant.

Reply:
Going to get reworked.